skiko/build.gradle.kts

```diff
@@ -915,6 +916,7 @@ fun createLinkJvmBindings(
                     "-static-libgcc",
                     "-lGL",
                     "-lX11",
+                    "-lXrandr",
```

Comment:
Please use dynamically loaded Xrandr, as in https://github.com/JetBrains/skiko/blob/4fb8aac1b4c277bdc4d1b14095f8fc5583f19f88/skiko/src/jvmMain/cpp/linux/display.cc#L22

Reply:
Done.